.github/CODEOWNERS

```diff
@@ -53,10 +65,10 @@ packages/storage-compat @maneesht @tonyjhuang @firebase/jssdk-global-approvers
 packages/storage-types @maneesht @tonyjhuang @firebase/jssdk-global-approvers
 
 # Messaging Code
-packages/messaging  @zwu52 @firebase/jssdk-global-approvers
-packages/messaging-compat  @zwu52 @firebase/jssdk-global-approvers
-packages/messaging-interop-types @zwu52 @firebase/jssdk-global-approvers
-integration/messaging @zwu52 @firebase/jssdk-global-approvers
+packages/messaging  @firebase/fcm-oncaller @firebase/jssdk-global-approvers
```

Comment:
Sorry, didn't see this until now but if you want to use a team here you have to create a team in the `firebase` organization in https://github.com/orgs/firebase/teams The comment doesn't do anything, it just clarifies who's in the already-existing team.